front-end/src/screens/Home/query.ts

```diff
@@ -17,6 +17,21 @@ export const QUERY_LATEST_POSTS = gql`
                     count
                 }
             }
+            type {
```

Comment:
if it's a post query it shouldn't have post in there. There must be a confusion here.

Reply:
Actually, this is also no longer used. -> Deleted it